packages/python/src/index.ts

```diff
@@ -208,7 +208,12 @@ export const build = async ({
     files: await glob('**', globOptions),
     handler: `${handlerPyFilename}.vc_handler`,
     runtime: pythonVersion.runtime,
-    environment: {},
+    environment: {
+      // This is required for environment variable access.
+      // In production, env var access is provided by static analysis
+      // so that only the used values are available.
+      ...(process.env as { [key: string]: string }),
```

Comment:
This should be a separate PR since its not related to Vite.

Also, do you have a failing test case that is solved by this change? It should be added to the test fixtures so we can verify this change fixes the issue.